src/customQueries.ts

```diff
@@ -28,3 +28,62 @@ export const customResponses = /* GraphQL */ `
     }
   }
 `;
+
+export const customGetUser = /* GraphQL */ `
```

Comment:
We'll want to execute this query for the User Context. Because of this, the User Context typings also need to be updated.